src/information-panel/information-panel.scss

```diff
@@ -187,3 +190,34 @@
     top: $iui-m * -0.5;
   }
 }
+
+/// Labels and inputs aligned using css grid
+@mixin iui-information-body-content {
+  .iui-input-label {
+    font-weight: normal;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  &:not(.iui-inline) {
+    // add bottom margin between pairs of label+input
+    > *:nth-child(even):not(:last-child) {
+      margin-bottom: 11px;
+    }
+  }
+
+  &.iui-inline {
+    display: grid;
+    row-gap: 11px;
+    grid-template-columns: auto 1fr;
+    align-items: center;
+    place-content: end;
+    text-align: end;
```

Comment:
This appears to be causing alignment issues in the non-editable information panel:
![Screen Shot 2021-11-09 at 1 00 21 PM](https://user-images.githubusercontent.com/849817/140979002-ed82b967-1c42-445e-bc81-c3499440f90f.png)